### DISCUSSION TITLE
Notification wrongly says "x requested your review"

### DISCUSSION THREAD

**Discussion by amadejpapez** [community/community#4229] [Mobile] [answered]
Hi! A review from another contributor was requested on a PR but notification in the app said "x requested _your_ review" which is wrong in this case. I am using the latest version of the app on iOS.

Maybe the issue is that a review from the owner of the repo was requested. I am added to it as a collaborator, so maybe that is causing this.

Here are the screenshots:

![image](https://user-images.githubusercontent.com/63184600/122830266-65f91380-d2e8-11eb-9e9c-cb51521b6f46.jpeg)

![image](https://user-images.githubusercontent.com/63184600/122830279-698c9a80-d2e8-11eb-99e8-6ae71e299e0c.jpeg)

**Comment by brookielee09** [comment]
> 🔐@
@$p3ct§©

©<Click or tap>🚧Emoji should be encrypted software...
Yall need to be careful 

Blocks are the fishbowl or some shit

€.3 
- _One_****

**Reply by soaud3322** [reply]
USA1015653 7834 User Perspot Download

**Reply by soaud3322** [reply]
soaud3322@gmail.com

**Comment by amrehman** [comment, marked as answer]
Hey there @amadejpapez thanks a lot for getting in touch.
That does sound like a bug, thanks for reporting. I have just created an issue for this on our engineering team and they will get to fix it asap!

**Reply by amadejpapez** [reply]
I am still experiencing this daily as I have notifications set on a few repos. Almost every review request causes notification preview to show "xyz requested _your_ review" even though he requested review from someone else.

Here is a screenshot from just a minute ago: 😕
![image](https://user-images.githubusercontent.com/63184600/136069440-dc1ed06f-b81b-49cb-8976-de9c303b57f1.jpeg)

**Reply by Fb9027** [reply]
Ok

**Comment by MilosPaunovic** [comment]
Hello, just wondering if there is any progress on this? I'm still experiencing it on Android application `v1.52.0 (640)`.

**Reply by haykam821** [reply]
I can also reproduce this on iOS with version 1.98.0.

**Reply by Fb9027** [reply]
> Hello, just wondering if there is any progress on this? I'm still experiencing it on Android application `v1.52.0 (640)`.

**Comment by Fb9027** [comment]
I did software update on my phone
